pyspecProcScripts/calc_error.py

```diff
@@ -0,0 +1,131 @@
+import pyspecdata as psd
+from .simple_functions import select_pathway
+import numpy as np
+
+
+# {{{ Functions to calculate var and mean of nan-masked data
+#     Pulled from time_domain_noise.py example
+def masked_mean_multi(x, axis=None):
```

Comment:
both this and the following function should be renamed starting with a single underscore, indicating they are intended to be private to this module.

Reply:
checked